docs/pages/web/toast.js

```diff
@@ -220,6 +209,98 @@ If  confirmation toast's text with more complex style is required, such as bold
 </Flex>`}
           />
         </MainSection.Subsection>
+
+        <MainSection.Subsection title="Dismissable">
```

Comment:
Instead of digging the hole deeper, can we create these new examples using Sandpack?

Reply:
This is coming afterwards https://github.com/pinterest/gestalt/pull/2551 I'll do that in that one